app/controllers/deploys_controller.rb

```diff
@@ -55,11 +55,11 @@ def search
     end
 
     if params[:deployer].present?
-      users = User.where("name LIKE ?", "%#{ActiveRecord::Base.send(:sanitize_sql_like, params[:deployer])}%").pluck(:id)
+      users = User.where("users.name LIKE ?", "%#{ActiveRecord::Base.send(:sanitize_sql_like, params[:deployer])}%").pluck(:id)
     end
 
     if params[:project_name].present?
```

Comment:
FYI I'd prefer a `if name = params[:project_name].presence` and then reuse name ... but either way is fine